app/_locales/fr/messages.json

```diff
@@ -50,5 +50,13 @@
   "InvalidAPI": {
     "message": "API Invalide, le service a peut être changer prévenez le responsable de cette extension.",
     "description": "Notification message displayed when API is not valid"
+  },
+  "showFullArticle": {
+    "message" : "Voir l'article complet",
+    "description": "Button added in all article to get the full content of the article"
+  },
+  "showArticlePreview": {
+    "message" : "Retourner à la prévisualisation",
+    "description": "Button added in all article to get the full content of the article"
```

Comment:
Same comment as above. The description here should be something like:

> Button added in all article to show the preview of the article

I also wonder if the description needs to be duplicated in all the translations, but for now I would leave it there :)